docs/core/docker/build-container.md

```diff
@@ -122,205 +135,194 @@ Counter: 4
 
-From the *app* folder, get a directory listing of the publish folder to verify that the *myapp.dll* file was created.
+```powershell
+dir .\bin\Release\netcoreapp3.1\publish\
 
-```console
-> dir bin\Release\netcoreapp3.1\publish
+    Directory: C:\Users\dapine\App\bin\Release\netcoreapp3.1\publish
 
-    Directory:  C:\docker-working\app\bin\Release\netcoreapp3.1\publish
 
-01/09/2020  11:41 AM    <DIR>          .
-01/09/2020  11:41 AM    <DIR>          ..
-01/09/2020  11:41 AM               407 myapp.deps.json
-01/09/2020  12:15 PM             4,608 myapp.dll
-01/09/2020  12:15 PM           169,984 myapp.exe
-01/09/2020  12:15 PM               736 myapp.pdb
-01/09/2020  11:41 AM               154 myapp.runtimeconfig.json
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----        4/27/2020   8:27 AM            434 NetCore.Docker.deps.json
+-a----        4/27/2020   8:27 AM           6144 NetCore.Docker.dll
+-a----        4/27/2020   8:27 AM         171520 NetCore.Docker.exe
+-a----        4/27/2020   8:27 AM            860 NetCore.Docker.pdb
+-a----        4/27/2020   8:27 AM            154 NetCore.Docker.runtimeconfig.json
 ```
 
-If you're using Linux or macOS, use the `ls` command to get a directory listing and verify that the *myapp.dll* file was created.
+#### [Linux](#tab/linux)
+
+Use the `ls` command to get a directory listing and verify that the *NetCore.Docker.dll* file was created.
 
 ```bash
 me@DESKTOP:/docker-working/app$ ls bin/Release/netcoreapp3.1/publish
-myapp.deps.json  myapp.dll  myapp.pdb  myapp.runtimeconfig.json
+NetCore.Docker.deps.json  NetCore.Docker.dll  NetCore.Docker.pdb  NetCore.Docker.runtimeconfig.json
 ```
 
 ## Create the Dockerfile
 
 The *Dockerfile* file is used by the `docker build` command to create a container image. This file is a text file named *Dockerfile* that doesn't have an extension.
 
-In your terminal, navigate back one directory to the working folder you created at the start. Create a file named *Dockerfile* in your working folder and open it in a text editor. Depending on the type of application you're going to containerize, you'll choose either the ASP.NET Core runtime or the .NET Core runtime. When in doubt, choose the ASP.NET Core runtime, which includes the .NET Core runtime. This tutorial will use the ASP.NET Core runtime image, but the application created in the previous sections is an .NET Core application.
-
-- ASP.NET Core runtime
+Create a file named *Dockerfile* in directory containing the *.csproj* and open it in a text editor. This tutorial will use the .NET Core runtime image, to correspond with the .NET Core console application.
 
-  ```dockerfile
-  FROM mcr.microsoft.com/dotnet/core/aspnet:3.1
-  ```
-
-- .NET Core runtime
-
-  ```dockerfile
-  FROM mcr.microsoft.com/dotnet/core/runtime:3.1
-  ```
+```dockerfile
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1
```

Comment:
The product group wants to push aspnet runtime as the runtime of choice as its the most compatible of the two (since it includes asp.net core stuff).

And, with the initial WARNING I added, and this section that points to ASP.NET (but leaves the option for non-ASP) the amount of complaints and bad feedback of "can't get my app to work" have pretty much disappeared. A lot of people are using ASP.NET Core and end up here...

Reply:
This one I'd like to push back on. We emphasize that if they're doing ASP.NET Core, there is a different article for that and that this article is explicitly using .NET Core. Without that, our docs would have zero examples of using the `runtime` image - if that is the case, why bother having it at all? I believe that this is the most appropriate place for this usage. Perhaps a compromise would be to include an alert letting them know about the aspnet image?